pep-0709.rst

```diff
@@ -126,13 +131,36 @@ the need to place these variables in a cell, allowing the comprehension (and all
 other code in the outer function) to access them as normal fast locals instead.
 This provides further performance gains.
 
+Module-level comprehensions are not inlined
+-------------------------------------------
+
 This PEP only proposes to inline comprehensions occurring inside functions,
 where fast-locals (``LOAD_FAST/STORE_FAST``) are used. Module-level
 comprehensions will continue to create and call a function. Module-level
 comprehensions are generally called only once (when the module is imported), so
-optimizing the performance of calling them is unnecessary, and inlining them
-would require separate code paths in the compiler to handle a module global
-namespace dictionary instead of fast-locals.
+optimizing their performance is low priority. Inlining them would require
+separate code paths in the compiler to handle a module global namespace
+dictionary instead of fast-locals, and it would be difficult or impossible to
+avoid breaking semantics, since the comprehension iteration variable itself
+would be a module global which might be referenced inside other functions,
+which might be called within the comprehension.
```

Comment:
Seems like there's an editing issue here with the double "which" that makes the intended semantics unclear.

Based on @JelleZijlstra 's comment, I believe what you intend to mean is:

```suggestion
would be a module global which might be referenced inside other functions
that are in turn called within the comprehension.
```